domains/somegiy.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Someguy154",
+           "email": "",
+           "discord": "1256385383139377193",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.ebnUO9YuEKLqnjlHGKJ9G2-On_ysqjYLViQC9LmYkbrc-tVzgSRrJ2s03T9UK90dksH6MSZudTYK0e2mb4st2CJEg9h0_J6TPMHNohQIwrCFKYw6IZJW_Skm_vj7Ue9qx89xTr9L7H5HzwjNTK9HECccmrUKBMBIRwwpTTtcEyoGrW8xX16swAio0ujDfSE3shomdJgo82k7s5ct5nrZYqMKkWWLz0nHfroevzUW_2Bwk3YQcs0G1FVNmxDA2ECYvUKID7RGAV1JyntvA0dnqU7XxcokBWosuiPJGdsF36M2e4W7xwgt8WGu0uYTj12aFAagPRNqyE8DnrB_Oqc0PQ.el0lMs_G29gYWVN2eKeEcA.Y_fyo6QOt4CJ82YJ2cwivAPmlZ5cnOREbVn-5KQLChrJJqaVXitX6BHCXFI-wCzsQtjeSHWAQgI80Z14xOFU9wdQ3JF1CamomfsNkof3qOQ._9OfbVmqcMH59Gb3gNELjw"
+        },
+    
+        "record": {
+            "CNAME": "https://someguy154.github.io/someguy/"
```

Comment:
`CNAME` record must contain a domain only, it cannot contain "`https://`" and "`/`".
You must change it to `someguy154.github.io`.